doc/source/bindings.rst

```diff
@@ -242,37 +245,56 @@ These are variables that behave like boolean flags. They can be True or
-CHUTES         yes          yes       Are the parachutes extended? (Treats all parachutes as one single unit. Does not activate them individually.)
-LIGHTS         yes          yes       Are the lights on? (like the "U" key in manual flight.)
-PANELS         yes          yes       Are the solar panels extended? (Treats all solar panels as one single unit. Does not activate them individually.)
-BRAKES         yes          yes       Are the brakes on?
-ABORT          yes          yes       Abort Action Group.
-AG1            yes          yes       Action Group 1.
-AG2            yes          yes       Action Group 2.
-AG3            yes          yes       Action Group 3.
-AG4            yes          yes       Action Group 4.
-AG5            yes          yes       Action Group 5.
-AG6            yes          yes       Action Group 6.
-AG7            yes          yes       Action Group 7.
-AG8            yes          yes       Action Group 8.
-AG9            yes          yes       Action Group 9.
-AG10           yes          yes       Action Group 10.
-AGn            yes          yes       If you have the Action Groups Extended mod installed, you can access its groups the same way, i.e. AG11, AG12, AG13, etc.
-============== ==========   ========= ===============
+Setting a value to these flags will result in actions triggered only if the value is changed.
+
+**NOTE ABOUT kOS CUSTOM ACTION GROUP FLAGS:** If the boolean flag is for a kOS action
+group, be aware that the value of the flag represents the actual state of the parts of the particular type
+(most of them will always return false if no corresponding parts found) 
+and is neither guaranteed to change immediately when a value is set, nor prevents the actions happening
+when the value is set to its current value (if the parts are in different states). 
+Setting the values to these fields will trigger all the corresponding parts that have different state than the one being set. 
+
+
+===================================================================  ==========   =========   ========= ===============
+Variable Name                                                         Can Read     Can Set     Source    Description
+===================================================================  ==========   =========   ========= ===============
+`SAS <commands/flight/systems.html#global:SAS>`__                     yes          yes          stock     (Same as "SAS" indicator on the navball.)
+`RCS <commands/flight/systems.html#global:RCS>`__                     yes          yes          stock     (Same as "RCS" indicator on the navball.)
+`GEAR <commands/flight/systems.html#global:GEAR>`__                   yes          yes          stock     Is the GEAR enabled right now? (Note, since it may be true with all or most gear retracted, you may want to set it off and back on to guarantee everything is deployed).
```

Comment:
I think I'm going to shorten these description entries before merging.  They're fairly long for the table, and get clipped do to width limits anyways.  The detailed information is available at the linked location anyways.